### DISCUSSION TITLE
Getting the latest version on Debian 12

### DISCUSSION THREAD

**Discussion by reafeichtinger** [cockpit-project/cockpit#19754] [Q&A] [answered]
Hello, first of all thanks for this amazing tool! Now to my question:

I installed cockpit via the provided guide for Debian in a Proxmox container but it seems like the installed version is 286-1.

![image](https://github.com/cockpit-project/cockpit/assets/80401865/5577f858-75c0-4b13-aa02-c93a4ff492ec)

The problem is that I would like to use the wireguard features added in 306 but I cannot figure out how to install the latest version.
I didn't use the backports repository since im on latest Debian and it shows the same version in the [backports](https://backports.debian.org/changes/bullseye-backports.html).
If there is a way to build from source as well I can try that but I need to migrate my current settings.

I'm grateful for any help, greets.

**Comment by martinpitt** [comment, marked as answer]
bullseye-backports has version 306, see the [package tracker](https://tracker.debian.org/pkg/cockpit). So you can enable backports like described in https://cockpit-project.org/running.html#debian
